secure-mobile-push/smp-android-client/securemobilepush/src/main/java/de/adorsys/android/securemobilepush/sms/SmsConfig.java

```diff
@@ -0,0 +1,12 @@
+package de.adorsys.android.securemobilepush.sms;
+
+import android.support.annotation.NonNull;
+
+class SmsConfig {
+    @NonNull
```

Comment:
Fixed in commit:https://github.com/adorsys/secure-key-storage/pull/44/commits/eb220e2d2982822d17e2173262b0ad7e0fb3e81c